documentation/dat/Card.dat.md

```
@@ -0,0 +1,56 @@
+# Card.dat
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)